WordPressUI/Ghosts/GhostCellDelegate.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public protocol GhostCellDelegate {
+    func willStartGhostAnimation()
```

Comment:
*VERY Nipticky* (apologies sorry i didn't notice before!). "Apple's standard" would probably be in the lines of:

- ghostAnimationWillStart
- ghostWillStartAnimating

The SDK's API tends to start with `who` + `what`. For instance:

- viewDidLoad / viewWillAppear / viewWill...
- tableView(_, cellForRowAtIndexPath)

Second note is that: instead of having the `cell` receive the event (which is, per se, never registered as a **delegate**), how about if:

```
func collectionView(_ collectionView: UICollectionView, willStartGhostAnimationAt indexPath: IndexPath)
func tableView(_ tableView: UITableView, willStartGhostAnimationAt indexPath: IndexPath)
```

That way these new delegate methods blend with **UITableViewDelegate** and 
**UICollectionViewDelegate**.  (Perhaps **GhostCollectionViewDelegate** and **GhostTableViewDelegate**?)

Thoughts?

Reply:
Third idea:

```
protocol GhostableView {
   func ghostAnimationWillStart()
}
```

Reply:
@jleandroperez no problem at all, please keep sending your feedback. 🥇 

I've applied your third idea.

I really liked the idea of blending with the `UITableViewDelegate` and `UICollectionViewDelegate` but we will end up doing exactly what `ghostAnimationWillStart` does now with less code.

Not that less code means better code, but in this case, it seems to me that the cell itself should be the one to be notified of an upcoming animation.

Let me know what you think. In any case, we can implement `GhostCollectionViewDelegate` and `GhostTableViewDelegate` in the future so Ghost will be even more flexible. 💯 

What do you think?